site/content/docs/user/working-offline.md

```diff
@@ -158,7 +158,7 @@ EOF
 Note that in an offline environment, in addition to preparing the node image,
 you also need to prepare HAProxy image in advance.
 
-Currently KIND uses `kindest/haproxy:2.1.1-alpine`,
+Currently KIND uses `kindest/haproxy:2.2.0-alpine`,
```

Comment:
this is out of date and we probably need a way to fix that. we use date-sha tags for all images/

Reply:
Or can I directly state that it can be found here? https://github.com/kubernetes-sigs/kind/blob/master/pkg/cluster/internal/loadbalancer/const.go#L20

Reply:
if most use cases don't mind the underlying haproxy version, there is also the option to not mention the exact version in the docs.

Reply:
great! let me change it.

Reply:
updated, PTAL. 
thanks!